ext/libxml2.cmd

```diff
@@ -0,0 +1,12 @@
+: # If you want to use a local build of libxml2, you must clone the libxml2 repo in this directory first, then enable CMake's AVIF_LOCAL_LIBXML2 option.
+: # The git tag below is known to work, and will occasionally be updated. Feel free to use a more recent commit.
+
+: # The odd choice of comment style in this file is to try to share this script between *nix and win32.
+
+git clone -b v2.11.5 --depth 1 https://gitlab.gnome.org/GNOME/libxml2.git
+
+mkdir -p libxml2/build.libxml2
```

Comment:
Note: Other `ext/` scripts use "build.libavif" naming for build folders. "build.libxml2" is fine too.